--- chatwoot-base/config/routes.rb	2026-03-24 00:48:29.569578534 +0000
+++ MenuPdvDmais/core-patches/config/routes.rb	2026-02-26 16:45:52.056584186 +0000
@@ -218,7 +218,6 @@
             delete :avatar, on: :member
             post :sync_templates, on: :member
             get :health, on: :member
-            post :register_webhook, on: :member
             if ChatwootApp.enterprise?
               resource :conference, only: %i[create destroy], controller: 'conference' do
                 get :token, on: :member
@@ -349,9 +348,7 @@
               post :send_instructions
               get :ssl_status
             end
-            resources :categories do
-              post :reorder, on: :collection
-            end
+            resources :categories
             resources :articles do
               post :reorder, on: :collection
             end
@@ -661,4 +658,9 @@
   # ----------------------------------------------------------------------
   # Routes for testing
   resources :widget_tests, only: [:index] unless Rails.env.production?
+
+  # Loading Enterprise Routes
+  if Rails.root.join('enterprise/config/routes.rb').exist?
+    instance_eval(File.read(Rails.root.join('enterprise/config/routes.rb')))
+  end
 end
